cmd/lora-app-server/main.go

```diff
@@ -174,7 +174,7 @@ func setRedisPool(c *cli.Context) error {
 }
 
 func setHandler(c *cli.Context) error {
-	h, err := mqtthandler.NewHandler(c.String("mqtt-server"), c.String("mqtt-username"), c.String("mqtt-password"), c.String("mqtt-ca-cert"))
+	h, err := mqtthandler.NewHandler(c.String("mqtt-server"), c.String("mqtt-username"), c.String("mqtt-password"), c.String("mqtt-ca-cert"), c.String("mqtt-cert"), c.String("mqtt-cert-key"))
```

Comment:
`c.String("mqtt-cert"), c.String("mqtt-cert-key")`

That should be `mqtt-tls-cert` an `mqtt-tls-key` I think?

Reply:
Uh oh, I missed... Sorry.